src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -223,239 +299,262 @@ def generateBaseMesh(cls, region, options):
 
             # Create nodes
             nodeIdentifier = 1
-            lNodeIds = []
-            uNodeIds = []
+            lowerNodeIdsLeft = []
+            upperNodeIdsLeft = []
+            lowerNodeIdsRight = []
+            upperNodeIdsRight = []
```

Comment:
I'd prefer these to be lowereLeftNodeIds, upperLeftNodeIds, lowerRightNodeIds, upperRightNodeIds.